docs/content/manual/pre-release/node-not-ready/kubelet-restart/kubelet-restart-on-a-node.md

```diff
@@ -5,54 +5,30 @@ title: Test kubelet restart on a node of the cluster
 ### Related issues:
 https://github.com/longhorn/longhorn/issues/2629
 
-### Case 1: Kubelet restart on RKE1 multi node cluster:
-1. Create a RKE1 cluster with config of 1 etcd/control plane and 3 worker nodes.
-2. Deploy Longhorn on the cluster.
-3. Deploy prometheus monitoring app on the cluster which is using Longhorn storage class or deploy a statefulSet with Longhorn volume.
-4. Write some data into the mount point and compute the md5sum.
-5. Restart the kubelet on the node where the statefulSet or Prometheus pod is running using the command `sudo docker restart kubelet`
-6. Observe the volume. It becomes degraded but is still running. 
-7. Once the node is back, the volume of the workload should work fine and the data is intact.
-8. Scale down then re-scale up the workload. Verify the existing data is correct.
-
-### Case 2: Kubelet restart on one node RKE1 cluster:
-1. Create a RKE1 cluster with config of 1 node all role.
-2. Deploy Longhorn on the cluster.
-3. Deploy prometheus monitoring app on the cluster which is using Longhorn storage class or deploy a statefulSet with Longhorn volume.
-4. Write some data into the mount point and compute the md5sum.
-5. Restart the kubelet on the node using the command `sudo docker restart kubelet`
-6. Check the instance manager pods on the node are still running.
-7. Observe the volume. It gets detached and the pod gets terminated (since the only replica of the volume becomes failed).
```

Comment:
The original test case includes testing the behavior of a single-replica volume. Could we also include it in robot test?

Reply:
Updated!